Essentials/src/com/earth2me/essentials/Console.java

```diff
@@ -74,4 +75,9 @@ public CommandSender getCommandSender() {
     @Override public void setReplyRecipient(IMessageRecipient recipient) {
         this.messageRecipient.setReplyRecipient(recipient);
     }
+
+    @Override
+    public boolean isHiddenFrom(Player player) {
+        return false;
```

Comment:
Doesn't seem to follow other code, where `@Override` and declarations are on the same line

Reply:
it's shitty codestyle from 1950, if we do a codestyle update again this will be addressed

Reply:
This is addressed in #3701